### PR TITLE
Show different warning on Order Creation if "Send to Patient"

### PR DESCRIPTION
**Pull request by captDaylight**
https://www.notion.so/photons/Warning-for-sending-order-doesn-t-make-sense-in-ROUTING-c6302e242160405f9a1b9b4779f07e59?pvs=4

### CODE REVIEW

**Review by samkotlove — APPROVED** [review]

File: apps/app/src/views/routes/NewOrder/components/OrderForm.tsx [hunk 1]
```diff
@@ -218,7 +218,10 @@ export const OrderForm = ({
 
         if (
           await confirmWrapper('Send Order?', {
-            description: 'This order will be immediately sent to the pharmacy of choice.',
+            // determine if order is "Send to Patient" by checking if pharmacyId is set
+            description: values.pharmacyId
+              ? 'This order will be sent immediately to the pharmacy of choice.'
+              : "This order will await the patient's choice of pharmacy.",
```
Comment (marisbest2) [inline apps/app/src/views/routes/NewOrder/components/OrderForm.tsx:224]: "This order will be sent when the patient chooses a pharmacy"

Maybe? 🤷

File: apps/app/src/views/routes/NewOrder/components/OrderForm.tsx [hunk 2]
```diff
@@ -218,7 +218,10 @@ export const OrderForm = ({
 
         if (
           await confirmWrapper('Send Order?', {
-            description: 'This order will be immediately sent to the pharmacy of choice.',
+            // determine if order is "Send to Patient" by checking if pharmacyId is set
+            description: values.pharmacyId
```
Comment (marisbest2) [inline apps/app/src/views/routes/NewOrder/components/OrderForm.tsx:222]: Should we determine the message based `pharmacyId` vs however we know that its a "send to patient"? 

🤷

**Review by marisbest2 — APPROVED** [review]
made an alternate sugestion, but looks good to me

File: apps/app/src/views/routes/NewOrder/components/OrderForm.tsx (hunk 1 above)
Comment (captDaylight) [reply apps/app/src/views/routes/NewOrder/components/OrderForm.tsx:224]: yeah that's better, adding!

File: apps/app/src/views/routes/NewOrder/components/OrderForm.tsx (hunk 2 above)
Comment (captDaylight) [reply apps/app/src/views/routes/NewOrder/components/OrderForm.tsx:222]: That's currently how we determine if an order is being sent to the patient, the `createOrder` mutation is implicitly missing `pharmacyId`

**Pull request merged**